src/pages-and-resources/discussions/app-list/AppCard.test.jsx

```diff
@@ -1,17 +1,84 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { render, screen } from '@testing-library/react';
+import { Factory } from 'rosie'; // eslint-disable-line import/no-extraneous-dependencies
+
+import AppCard from './AppCard';
+import messages from './messages';
+import '../data/__factories__/index';
+
 describe('AppCard', () => {
+  const partailSupportApp = Factory.build('app');
```

Comment:
@asadazam93 ^ Rosie

Reply:
@awaisdar001 I think we should probably discuss before adding new packages